infrastructure/monitoring.py

```diff
@@ -1,12 +1,54 @@
-from pulumi_gcp import cloudrun, secretmanager
-from pulumi import FileAsset
+from pulumi_gcp import cloudrun, secretmanager, storage
+from pulumi import Config, FileAsset, Output
 import project
+import buckets
 
-prometheus_config_secret = secretmanager.Secret("prometheus-config")
-prometheus_config_version = secretmanager.SecretVersion(
-    "prometheus-config-version",
-    secret=prometheus_config_secret.id,
-    secret_data=FileAsset("infrastructure/prometheus.yaml"),
+config = Config()
+
+grafana_admin_password = config.require_secret("GRAFANA_ADMIN_PASSWORD")
+
+grafana_admin_password_secret = secretmanager.Secret(
+    "grafana-admin-password",
```

Comment:
Nitpick I like "administrator" but if you prefer "admin" that's okay.

Reply:
fixed